File: pinyougou-manager-web/src/main/java/com/pinyougou/manager/controller/SellerController.java
package com.pinyougou.manager.controller;

import com.pinyougou.pojo.TbSeller;
import com.pinyougou.sellergoods.service.SellerService;
import entity.PageResult;
import entity.Result;
import org.junit.Test;
import org.springframework.web.bind.annotation.RequestBody;
import org.springframework.web.bind.annotation.RequestMapping;
import org.springframework.web.bind.annotation.RestController;

import javax.annotation.Resource;

@RestController
@RequestMapping("/seller")
public class SellerController {
    @Resource
    private SellerService sellerService;

    @RequestMapping("/search")
    public PageResult search(@RequestBody TbSeller tbSeller,int page,int rows){
        return sellerService.search(tbSeller,page,rows);
    }
    @RequestMapping("/findOne")
    public TbSeller findOne(String sellerId){
        return sellerService.findOne(sellerId);
    }
    @RequestMapping("/updateStatus")
    public Result updateStatus(String sellerId,String status){
        try {
            sellerService.updateStatus(sellerId,status);
            return new Result(true,"审核成功");
        }catch (Exception e){
            return new Result(false,"审核失败");
        }
    }

}

File: pinyougou-manager-web/src/main/webapp/js/controller/sellerController.js
 //控制层 
app.controller('sellerController' ,function($scope,$controller,sellerService){

	$controller('baseController',{$scope:$scope});//继承
    $scope.search=function(searchEntity,page,rows){
        sellerService.search(searchEntity,page,rows).success(function (response) {
            $scope.list = response.rows;
            $scope.paginationConf.totalItems=response.total;
        });
    }
    $scope.findOne = function (sellerId) {
        sellerService.findOne(sellerId).success(function (response) {
            $scope.entity = response;
        });
    }
    $scope.updateStatus  =function (sellerId,status) {
        sellerService.updateStatus(sellerId,status).success(function (response) {
            if(response.success){
                $scope.reloadList();
            }else{
                alert(response.message);
            }
        });
    }
});	
